deploy/TokenBridgeInit.sol

```diff
@@ -95,6 +99,7 @@ library TokenBridgeInit {
             targetData:  abi.encodeCall(L2TokenBridgeSpell.init, (
                 l2BridgeInstance.govRelay,
                 l2BridgeInstance.bridge,
+                l2BridgeInstance.bridgeImp,
```

Comment:
We probably want to also add the implementation key to the chainlog (at the end of this function), as done in the upgradeable tokens.

Reply:
Done